lib/jsonapi-response.js

```diff
@@ -0,0 +1,102 @@
+const attributesMap = require('../fixtures/attributes');
+const relationshipsMap = require('../fixtures/relationships');
+const config = require('../config');
+
+module.exports = (data) => {
+  const response = {};
+
+  response.data = {};
+  response.data.type = data._type;
+  response.data.id = data._id;
+  response.data.attributes = getAttributes(data._source);
+
+  response.relationships = getRelationships(data._source);
+
+  response.included = getIncluded(data._source, response);
+
+  response.links = {};
+  response.links.self = config.rootUrl + '/' + data._type + '/' + data._id;
+
+  return JSON.stringify(response);
+};
+
+function getAttributes (data, dataType) {
+  const attributes = {};
+
+  Object.keys(data).forEach((key) => {
+    if (attributesMap.indexOf(key) > -1) {
+      attributes[key] = data[key];
+    }
+  });
```

Comment:
It would be quicker to loop over `attributesMap` and add it to `attributes` if it exists on `data` - would that work or am I missing something?